pkg/sql/planhook.go

```diff
@@ -99,6 +100,7 @@ type PlanHookState interface {
 	CreateSchemaNamespaceEntry(ctx context.Context, schemaNameKey roachpb.Key,
 		schemaID descpb.ID) error
 	MigrationJobDeps() migration.JobDeps
+	BufferClientNotice(ctx context.Context, notice pgnotice.Notice)
```

Comment:
Are we actually using this?

Reply:
oops, not yet. Once the CLI issue is resolved we'll need it. Removing!